helm/app-operator/templates/rbac.yaml

```diff
@@ -51,30 +51,58 @@ rules:
     - create
     - patch
     - update
+- apiGroups:
```

Comment:
Additional rules are for creating the helm-2to3-migration job and related resources.

https://github.com/giantswarm/helm-2to3-migration/tree/bdac5f952e479ffc582d408818101a6309fda44a/helm/helm-2to3-migration/templates

Reply:
is it necessary? We created it by helm anyway, and myself never encountered with an RBAC error related to this chart.

Reply:
If you install the chart with tiller it has cluster-admin. Using helm 3 version of helmclient there is no tiller so app-operator needs the permissions.